ui/src/layouts/BuildPublication.tsx

```diff
@@ -353,14 +357,31 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
                     <div className="mb-12">{props.children}</div>
                     {/* bottom next and back nav buttons */}
                     <div className="flex justify-end space-x-8 ">
+                        <NavigationButton
```

Comment:
Is this preview button meant to be just on the bottom of the 'review' page, or on every step of the publication flow?

Reply:
Every step - and grayed out when the publication hasn't been fully filled out. Similar to the bar above the publication flow

Reply:
Ah okay then :) if a second preview button is being added to every publication step page then it might be good to update the acceptance criteria in this PR to match that as currently is says 'Add Preview button to bottom of review page'